valhalla/thor/service.h

```diff
@@ -130,6 +129,7 @@ class thor_worker_t {
   SOURCE_TO_TARGET_ALGORITHM source_to_target_algorithm;
   boost::optional<int> date_time_type;
   valhalla::meili::MapMatcherFactory matcher_factory;
+  valhalla::baldr::GraphReader& reader;
```

Comment:
'reader' should be declared after matcher_factory since it gets initialized from matcher_factory.